### DISCUSSION TITLE
How to use the fuzzy search?

### DISCUSSION THREAD

**Discussion by sangdth** [Budibase/budibase#7814] [Help] [answered]
In the `Search for rows` API, how can I use the `fuzzy`?

I've tried:
- Google, no results.
- Search here. All results: `"fuzzy": {}`
- Searched Budibase Docs, no results.

Can someone give a hint? Thanks 🙏🏻

**Comment by melohagan** [comment, marked as answer]
Hi @sangdth 

Fuzzy filter is specific to the internal DB which uses CouchDB. More on fuzzy queries can be found here: https://lucene.apache.org/core/4_3_0/queryparser/org/apache/lucene/queryparser/classic/package-summary.html#Fuzzy_Searches

**Reply by sangdth** [reply]
Thank you @melohagan , I can not try it right now but I will leave a comment when I have something. Have a nice day.

**Reply by melohagan** [reply]
You too! :)
